content/ghc-9.12-cabal-3.14/index.md

```diff
@@ -0,0 +1,128 @@
++++
+title = "GHC 9.12 & cabal 3.14 releases"
```

Comment:
Shouldn't it be either `Cabal` or `cabal-install`? Seems odd to choose lower case.